config.py

```diff
@@ -8,9 +8,9 @@
 SWAGGER_UI_DOC_EXPANSION = 'none'
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
-API_DESC = 'An API for serving models'
-API_VERSION = '0.1'
+API_TITLE = 'MAX Scene Classifier'
+API_DESC = 'Classify images according to the place/location labels in the Places365 data set.'
+API_VERSION = '1.0.0'
```

Comment:
Just curious, how are you deciding on the version number? I haven't been updating this on other models.

Reply:
We haven't been disciplined about this - really it should match the released version...

Here I guess this would need to be bumped to `1.1.0`